userchrome.css

```diff
@@ -30,3 +30,27 @@
 .titlebar-buttonbox-container {
   display: none !important;
 }
+
+#navigator-toolbox {
+  position: relative;
+  z-index: 1;
+  height: 20px;
+  margin-bottom: 0px;
+  transition: height 250ms ease-in-out, opacity 250ms ease-in-out;
+  opacity: 0 !important;
+  overflow: hidden;
+}
+
+#navigator-toolbox:hover {
+  height: 40px;
+  opacity: 1 !important;
+  transition: height 250ms ease-in-out, opacity 75ms ease-in-out;
+}
+
+#navigator-toolbox:focus-within {
+  height: 40px;
+  opacity: 1 !important;
+  transition: height 250ms ease-in-out, opacity 75ms ease-in-out;
+  overflow: visible !important;
+}
```

Comment:
LGTM!

Reply:
That's works flawlessly on mine, the animations are pretty sick too. 

Also, it also great if we add the on hover `tab bar` same as the `search bar` like as we did